demo_nodes_py/README.md

```diff
@@ -0,0 +1,280 @@
+## **What Is This?**
+
+This demo ...
+
+This package consists of ROS 2 nodes:
+
+1. `listener`
+2. `talker`
+3. `add_two_ints_client`
+4. `add_two_ints_server`
+5. `set_parameters_callback`
+6. `add_two_ints_client_async`
+7. `listener_qos`
+8. `talker_qos`
+9. `listerner_serialized`
+10. `async_param_client`
+
+## **Build**
+
+Run this command to build the package:
+
+```bash
+colcon build --packages-select demo_nodes_py
+```
+
+## **Run**
+
+### Basic Listener & Talker
+
+This runs `talker` and `listener` ROS 2 nodes which exchanges the following string with incremeting integer:
```

Comment:
"...which exchange the following..." , remove the extra 's'

Reply:
Done.